test/unit/associations/belongs-to-many.test.ts

```diff
@@ -23,25 +36,31 @@ describe(getTestDialectTeaser('belongsToMany'), () => {
     expect(() => {
       // @ts-expect-error
       User.belongsToMany();
-    }).to.throw('User.belongsToMany called with something that\'s not a subclass of Sequelize.Model');
+    }).to.throw(
+      'User.belongsToMany called with undefined that\'s not a subclass of Sequelize.Model',
```

Comment:
Same as my other comments, but I'll still do it so I don't have to go through all styling changes again

Reply:
btw. not sure if our linter will allow this but feel free to use backticks instead of single quotes to save the escaping inside of the string

Reply:
It should allow it